fgrpc/ping.proto

```diff
@@ -2,6 +2,7 @@
 // protoc ping.proto --go_out=plugins=grpc:.
 syntax = "proto3";
 package fgrpc;
+option go_package="../fgrpc";
```

Comment:
that looks very odd; we're already in fgrpc?

Reply:
This is a hack for now –  I don't know why my protoc makes me specify this, makes me use a `/` character. 

An alternative would be `go_package ./fgrpc` and then running the protoc from `..`

I haven't had a chance to clean this up or figure out the right way.

Reply:
Looks like specifying the fully qualified package + using source_relative is better.